mapper/mapper.go

```diff
@@ -31,30 +31,24 @@ func CreateStartPage(basePage coreModel.Page) StartPageModel {
 	return model
 }
 
-// FeatureFlags
-type FeatureFlags struct {
-	EnabledBreadcrumbs bool
-}
-
 type AreaModel struct {
 	coreModel.Page
-	Name               string           `json:"name"`
-	Level              string           `json:"level"`
-	Code               string           `json:"code"`
-	Ancestors          []areas.Ancestor `json:"ancestors"`
-	Siblings           []AreaModel      `json:"siblings"`
-	Children           []AreaModel      `json:"children"`
-	Relations          []areas.Relation `json:"relations"`
-	EnabledBreadcrumbs bool
+	Name      string           `json:"name"`
+	Level     string           `json:"level"`
+	Code      string           `json:"code"`
+	Ancestors []areas.Ancestor `json:"ancestors"`
```

Comment:
Super minor but we have aliased the package "github.com/ONSdigital/dp-api-clients-go/v2/areas" as `areas` but the package is already called `areas` so don't think we need to alias it

Reply:
`areas` names package declaration removed on line 6